package.json

```diff
@@ -31,33 +31,14 @@
     "@angular/forms": "^4.3.6",
     "@angular/router": "^4.3.6",
     "@skyux/core": "^3.1.0",
-    "@skyux/i18n": "^3.2.0",
+    "@skyux/i18n": "^3.3.0",
     "@skyux/indicators": "^3.0.0-rc.1",
     "@skyux/popovers": "^3.0.0-rc.0"
   },
   "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "4.3.6",
-    "@angular/common": "4.3.6",
-    "@angular/core": "4.3.6",
-    "@angular/forms": "4.3.6",
-    "@angular/http": "4.3.6",
-    "@angular/platform-browser": "4.3.6",
-    "@angular/router": "4.3.6",
     "@blackbaud/skyux": "2.26.0",
```

Comment:
Mind bumping skyux to latest so that we automatically pull in the latest versions of the modules?